<!-- trajectory recorder launch -->
<launch >
	<!-- arguments -->
	<arg name="path_file" default="$(find trajectory_tools)/../../maps/nic_garage/sample_path.txt" />

	<!-- trajectory recorder -->
	<node name="trajectory_recorder" pkg="trajectory_tools" type="trajectory_recorder" output="screen" >
		<param name="/map_frame" value="/world" type="string" />
		<param name="/base_link_frame" value="/base_link" type="string" />
		<param name="/path_file_name" value="$(arg path_file)" type="string" />
		<param name="/dist_interval" value="0.5" type="double" />
		<param name="/angle_interval" value="2.0" type="double" />
	</node >
</launch >
